anomaly_utilities.py

```diff
@@ -228,7 +228,7 @@ def xfade(xfor, xbac):
     return x
 
 
-def set_dynamic_threshold(data, alpha, window_sz):
+def set_dynamic_threshold(residuals, alpha, window_sz):
     """Determines a threshold based on the local confidence interval, 
     considering the data looking forward and backward window_sz steps.
     data is a series like object
```

Comment:
update "data" to be "residuals"

Reply:
updated the comments in the function documentation